pkg/cmd/integration.go

```diff
@@ -107,12 +108,12 @@ func (bc *IntegrationCmd) CreateIntegrationCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "integration <consuming_service_instance_id> <providing_service_instance_id>",
 		Short: "integrate certain mobile services together",
-		Long: `example usage: kubectl plugin mobile create integration <consuming_service_instance_id> <providing_service_instance_id>
```

Comment:
I think in the long description we should give an example of how to get the serviceinstance ids

``` 
mobile get serviceinstances fh-sync-server
mobile get serviceinstances keycloak
```